cmd/solvercli/cmd/balances.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/skip-mev/go-fast-solver/shared/clients/skipgo"
+	"github.com/skip-mev/go-fast-solver/shared/config"
+	"github.com/skip-mev/go-fast-solver/shared/lmt"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+type SkipBalancesRequest struct {
+	Chains map[string]ChainRequest `json:"chains"`
+}
+
+type ChainRequest struct {
+	Address string   `json:"address"`
+	Denoms  []string `json:"denoms"`
+}
+
+type SkipBalancesResponse struct {
+	Chains map[string]ChainResponse `json:"chains"`
+}
+
+type ChainResponse struct {
+	Address string                 `json:"address"`
+	Denoms  map[string]DenomDetail `json:"denoms"`
+}
+
+type DenomDetail struct {
+	Amount          string `json:"amount"`
+	Decimals        uint8  `json:"decimals"`
+	FormattedAmount string `json:"formatted_amount"`
+	Price           string `json:"price"`
+	ValueUSD        string `json:"value_usd"`
+}
+
+type ChainBalance struct {
+	ChainID    string
+	AssetDenom string
+	Balance    *big.Int
+	Symbol     string
+	Decimals   uint8
+	PriceUSD   *big.Float
+	ValueUSD   *big.Float
+}
+
+type ChainGasBalance struct {
+	ChainID           string
+	Balance           *big.Int
+	Symbol            string
+	Decimals          uint8
+	WarningThreshold  *big.Int
+	CriticalThreshold *big.Int
+}
+
+var balancesCmd = &cobra.Command{
+	Use:   "balances",
```

Comment:
we should document the `--custom-asssets` flag in an area besides just the example usage. it took me a big to figure out where that was coming from in the code. I think we can put it in the `Use` field https://github.com/spf13/cobra/blob/v1.8.1/command.go#L51, or maybe the `Args`

Reply:
https://github.com/spf13/cobra/blob/v1.8.1/command.go#L51 actually this is probably the right way https://github.com/spf13/cobra/blob/main/site/content/user_guide.md#local-flags

Reply:
yessir i've already added an explanation there